app/src/main/kotlin/com/ark/globe/jsonprocess/JSONFile.kt

```diff
@@ -77,10 +66,9 @@ class JSONFile {
                             with(bufferedReader) {
                                 forEachLine {
                                     jsonLocation.append(it)
-                                    println("Line: $it")
                                 }
                                 locations.add(JSONParser.parseFromJsonToLocation(jsonLocation.toString()))
-                                println("File ${numberOfFiles++} Contents: $jsonLocation")
+                                Log.d("File ${numberOfFiles++}:",  jsonLocation.toString())
```

Comment:
I think, such code style should be avoided.. Yes, it's shorter than
```
numberOfFiles += 1
Log.d("File ${numberOfFiles}:", jsonLocation.toString())
```
but this snippet is better for readability.

Reply:
Noted.